clients/src/main/java/org/apache/kafka/common/metrics/stats/Min.java

```diff
@@ -39,7 +39,7 @@ public double combine(List<Sample> samples, MetricConfig config, long now) {
         double min = Double.MAX_VALUE;
         for (Sample sample : samples)
             min = Math.min(min, sample.value);
-        return min;
+        return Math.abs(min - Double.MAX_VALUE) < 0.001 ? Double.NaN : min;
```

Comment:
Spotbugs was complaining about comparing doubles (FE_FLOATING_POINT_EQUALITY)

Reply:
Yeah, it's a tricky business... I think the suggestion I had in Max would also apply here, and you wouldn't have to compare them at all.